src/devices/sound/swp00.h

```diff
@@ -43,6 +43,7 @@ class swp00_device : public device_t, public device_sound_interface, public devi
 	static const std::array<s32, 0x20> decay_linear_step;
 	static const std::array<s32, 16> panmap;
 	std::array<s32, 0x80> m_global_step;
+	static const std::array<u8, 4> m_dpcm_offset;
```

Comment:
You shouldn’t use `m_` prefixes for static data members, only instance members.  It’s also clearer to group static data members and instance data members rather than intermixing them.

Reply:
@cuavas Ah right... Will fix (but maybe I'll wait a bit, in case there are other issues). Thanks!

Reply:
@galibert is probably the best person to review the functional changes here.